src/NuGet.Clients/PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -166,6 +171,7 @@ public void ApplyShowPreviewSetting(bool show)
             UserSettings settings,
             Configuration.ISettings nugetSettings)
         {
+            var dependencySetting = GetDependencyBehaviorFromConfig(nugetSettings);
             if (settings == null)
             {
                 if (nugetSettings == null)
```

Comment:
This null check should be removed now, otherwise it looks like it would default to what the UI defaults to instead which is confusing.